feature/policy_forwarding/otg_tests/mpls_gre_ipv4_decap_scale_test/mpls_gre_ipv4_decap_scale_test.go

```diff
@@ -0,0 +1,580 @@
+			r = append(r, label)
+			label += mplsLabelStep
+		}
+		return r
+	}()
+
+	var interfaces []*otgconfighelpers.InterfaceProperties
+
+	for i := range intCount {
+		iface := &otgconfighelpers.InterfaceProperties{
+			Name:        fmt.Sprintf("agg1port%d", i+1),
+			IPv4:        netConfig.otgIPs[i],
+			IPv4Gateway: netConfig.dutIPs[i],
+			Vlan:        uint32(i + 1),
+			IPv4Len:     30,
+			IPv6:        netConfig.otgIPsV6[i],
+			IPv6Gateway: netConfig.dutIPsV6[i],
+			IPv6Len:     126,
+			MAC:         netConfig.otgMACs[i],
+		}
+		interfaces = append(interfaces, iface)
+	}
+
+	agg1.Interfaces = interfaces
+	// Get default parameters for OC Policy Forwarding
+	ocPFParams := fetchDefaultOcPolicyForwardingParams()
+
+	flowOuterIPv4Validation.Interface.Names = append(flowOuterIPv4Validation.Interface.Names, agg1.Interfaces[0].Name)
+	flowOuterIPv4.RxNames = append(flowOuterIPv4.RxNames, agg1.Interfaces[0].Name+".IPv4")
+	flowOuterIPv6Validation.Interface.Names = append(flowOuterIPv6Validation.Interface.Names, agg1.Interfaces[0].Name)
+	flowOuterIPv6.RxNames = append(flowOuterIPv6.RxNames, agg1.Interfaces[0].Name+".IPv4")
+
+	for i, iface := range agg1.Interfaces {
+		// Limiting it to 50 since checking ARP for 2000 interfaces takes long time
+		if i >= 50 {
+			break
+		}
+		flowResolveArp.Interface.Names = append(flowResolveArp.Interface.Names, iface.Name)
+	}
+	configureOTG(t)
+
+	// Pass ocPFParams to ConfigureDut
+	ocPFParams.DecapPolicy.MplsStaticLabels = mplsStaticLabels
+	ocPFParams.DecapPolicy.MplsStaticLabelsForIpv6 = mplsStaticLabelsForIpv6
+	configureDut(t, dut, netConfig, ocPFParams)
+
+}
+
+// fetchDefaultOcPolicyForwardingParams provides default parameters for the generator,
+// matching the values in the provided JSON example.
+func fetchDefaultOcPolicyForwardingParams() cfgplugins.OcPolicyForwardingParams {
+	return cfgplugins.OcPolicyForwardingParams{
+		NetworkInstanceName: "DEFAULT",
+		InterfaceID:         "Agg1.10",
+		AppliedPolicyName:   "customer1",
+	}
+}
+
+// function should also include the OC config , within these deviations there should be a switch statement is needed
+// Modified to accept pf, ni, and ocPFParams
+func decapMPLSInGRE(t *testing.T, dut *ondatra.DUTDevice, pf *oc.NetworkInstance_PolicyForwarding, ni *oc.NetworkInstance, netConfig *networkConfig, ocPFParams cfgplugins.OcPolicyForwardingParams) {
+	ocPFParams.DecapPolicy.NextHops = netConfig.otgIPs
+	ocPFParams.DecapPolicy.NextHopsV6 = netConfig.otgIPsV6
+	ocPFParams.DecapPolicy.ScaleStaticLSP = true
+	cfgplugins.MplsConfig(t, dut)
+	cfgplugins.QosClassificationConfig(t, dut)
+	cfgplugins.LabelRangeConfig(t, dut)
+	cfgplugins.DecapGroupConfigGre(t, dut, pf, ocPFParams)
+	cfgplugins.MPLSStaticLSPConfig(t, dut, ni, ocPFParams)
+	if !deviations.PolicyForwardingOCUnsupported(dut) {
+		pushPolicyForwardingConfig(t, dut, ni)
+	}
+}
+
+// PF-1.13.2: Verify IPV4/IPV6 traffic scale
+func TestMPLSOGREDecapIPv4AndIPv6(t *testing.T) {
+	t.Helper()
```

Comment:
I think this should be removed.  This is an actual test function that should be tracked for pass/fail.  Not a helper, right?  

By calling t.Helper() at the beginning of a helper function, you instruct the Go test runner to skip that function when reporting file and line information for test failures. Instead, the reported location will be the line in the test function that called the helper, making it much clearer where the issue originated.